lib/dependency-manager-adapters/npm.js

```diff
@@ -7,12 +7,32 @@ var path = require('path');
 var extend = require('extend');
 var debug = require('debug')('ember-try:dependency-manager-adapter:npm');
 var rimraf = RSVP.denodeify(require('rimraf'));
+var execFileSync = require('child_process').execFileSync;
 
 module.exports = CoreObject.extend({
   init: function() {
     this._super.apply(this, arguments);
     this.run = this.run || require('../utils/run');
+    this.adapterInit();
   },
+  adapterInit: function() {
+    this.backupYarnLockPresent = fs.existsSync(path.join(this.cwd, this.yarnLockBackupFileName));
```

Comment:
I don't think this check makes sense here ... the backup `yarn.lock` would only exist here if the task were started from a dirty state, otherwise, the backup `yarn.lock` is moved into place by the `setup` method here.